lib/feedjira/parser/podcast.rb

```diff
@@ -19,6 +19,7 @@ class Podcast
       element :description
       element :language
       element :copyright
+      element :author
```

Comment:
ah, gotcha. OK, so import currently can't set author_email if it's only using `<itunes:author>`.